src/microbe_stage/SpawnSystem.cs

```diff
@@ -372,43 +383,40 @@ private int SpawnWithSpawner(Spawner spawnType, Vector3 location, ref int spawns
             return spawns;
         }
 
-        if (spawnType is CompoundCloudSpawner || estimateEntityCount < Settings.Instance.MaxSpawnedEntities)
```

Comment:
I see we already had this code, but I think cleaning things up now would be worth it. If you aren't fully up for it, I can do that refactor the way I like.